CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# dbt_twilio_source v0.2.0
+
+## Breaking Changes
```

Comment:
Can you add the red lights here to bring attention to the breaking change

Reply:
added!